changelog/unreleased/pr-13928.toml

```diff
@@ -0,0 +1,5 @@
+type = "fixed"
+message = "Test to prevent pipeline rule description bug (13885)"
```

Comment:
This point was made by @linuspahl in another PR and I think was a good reminder: We only need to add an entry to the changelog if the changes made in the PR affect the user - in this one it is not the case: https://graylogdocumentation.atlassian.net/wiki/spaces/GENERAL/pages/2518908931/Changelog+Entries#When-to-include-a-changelog-entry%3F

I think it would be good to remove this as I imagine someone reading the changelog would like to only see information important to them.

You could add this PR to the changelog of the original fix though if you want to track it.

Reply:
Yep I will remove it.